src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -249,6 +238,11 @@ Visitor::Triples Visitor::visitTypesafe(Parser::ConstructTemplateContext* ctx) {
                                  : Triples{};
```

Comment:
After thinking about this:
In the visitor here it should be `visitOptional`, because that's what we always do for
`ruleName : otherRuleName?`.

And then the calling code (you find it, because it breaks), make it
`Triples blubb = visitOrParserOrWhateverThisContext(..).value_or(Triples{});`